style.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: blueviolet;
+}
```

Comment:
- [ ] [OPTIONAL] Kindly change the `<h1>` element and use classes, it's a  good practice to use classes in the stylesheet instead of the element name.